mesonbuild/ast/interpreter.py

```diff
@@ -14,8 +14,11 @@
 
 # This class contains the basic functionality needed to run any interpreter
 # or an interpreter-based tool.
+from __future__ import annotations
+
+import os, sys
```

Comment:
one import per line, please :)

Reply:
:p and here I just moved existing blocks of code around.